pkg/server/router/credential.go

```diff
@@ -46,9 +46,10 @@ type CreateCredentialRequest struct {
 	// A context is optional. If not present, we'll apply default, required context values.
 	Context string `json:"@context"`
 	// A schema is optional. If present, we'll attempt to look it up and validate the data against it.
-	Schema string                 `json:"schema"`
-	Data   map[string]interface{} `json:"data" validate:"required"`
-	Expiry string                 `json:"expiry"`
+	Schema    string                 `json:"schema"`
+	Data      map[string]interface{} `json:"data" validate:"required"`
+	Expiry    string                 `json:"expiry"`
+	Revocable bool                   `json:"revocable"`
```

Comment:
This might be part of a bigger conversation, but do we add documentation to the fields that we're adding? API consumers will need to understand what every field does when they make a request, and not all are self explanatory.

Reply:
I'm not sure how this ties into the swagger generation we have today. It's a good note - would you mind opening an issue to improve the api docs?